_actions/assistant.py

```diff
@@ -217,7 +217,7 @@ def _install_repo(repo: Dict[str, str], remove_dir: bool = True) -> List[str]:
 
         flags = AssistantCLI._get_flags(repo)
         cmds.append("pip install " + " ".join([pip_install] + flags))
-        cmds.append("pip list")
+        # cmds.append("pip list")
```

Comment:
```suggestion
```

Reply:
this is rather debugging so lets keep it :)

Reply:
```suggestion
        cmds.append("pip list")
```